modules/auxiliary/gather/bluetooth_pair.rb

```diff
@@ -0,0 +1,67 @@
+##
+# This module requires Metasploit: http://metasploit.com/download
+# Current source: https://github.com/rapid7/metasploit-framework
+##
+
+require 'bluetooth'
+
+class MetasploitModule < Msf::Auxiliary
+
+  def initialize(info = {})
+    super(update_info(info,
+      'Name'           => 'Bluetooth Simple Pair',
+      'Description'    => %q{
+        This module simply pairs over Bluetooth with a target device.
+      },
+      'Author'         => [ 'Carter Brainerd <cbrnrd>' ],
+      'License'        => MSF_LICENSE
+    ))
+
+    register_options([
+      OptString.new('MAC_ADDR', [true, 'The MAC address to pair to'])
+      ])
+
+  end
+
+  VALID_MAC = /^([0-9a-fA-F]{2}[:-]){5}[0-9a-fA-F]{2}$/i
```

Comment:
Minor, but the case insensitivity modifier `i` removes the need to do `[0-9a-fA-F]` -- `[0-9a-f]` should suffice.